addons/project_todo/static/src/web/activity/activity_menu_patch.js

```diff
@@ -32,6 +36,18 @@ patch(ActivityMenu.prototype, "todo", {
             }
             this.state.addingTodo = false;
         });
+        useCommand(
+            this.env._t("Add a To-Do"),
+            () => {
+                document.body.click(); // hack to close command palette
+                this.createActivityTodo();
+            },
+            {
+                category: "smart_action",
```

Comment:
This is not really a smart action. Maybe it could be in its own category? Or with commands such as "Home menu"?